tmpfiles.d/systemd-nspawn.conf

```diff
@@ -5,7 +5,7 @@
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
-# See tmpfiles.d(5) for details
+# See tmpfiles.d(5) for details.
```

Comment:
BTW, this file should probably be renamed to systemd-machine, given we now have vmspawn that makes use of these dirs.

Reply:
Renaming of files is problematic… People can have masks or oerrides and those break when we change the name. So I think we should just keep the name, even if it doesn't fit exactly, unless there's a strong reason to rename.

Reply:
Indeed. Please ignore then.